src/SameAs.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3;
+
+class SameAs implements SameAsInterface
```

Comment:
Maybe this needs an extra suffix or prefix to make it more clear it has to do with uitinvlaanderen?